tests/smartclone_test.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]SmartClone tests",
 
 	Describe("Verify DataVolume Smart Cloning - Negative flow", func() {
 		It("verify inapplicable smart-clone dv", func() {
-			if controller.IsOpenshift(f.K8sClient) {
+			if !f.IsSnapshotStorageClassAvailable() {
```

Comment:
I'm confused why this check switched from checking for openshift to checking for the presence of a storage class.

Reply:
Basically with this PR, instead of checking for openshift (which at some point will have snapshotting as well) we check if the passed in snapshot storage class exists. If so, then we run the test.

This allows us to have the check by feature instead of openshift vs kubernetes where kubernetes is simply ahead by feature for a short period of time. This makes the test more robust.